src/app/containers/SocialEmbed/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useContext } from 'react';
+import path from 'ramda/src/path';
+import styled from 'styled-components';
+import {
+  AmpSocialEmbed,
+  CanonicalSocialEmbed,
+} from '@bbc/psammead-social-embed';
+import { GEL_SPACING_TRPL } from '@bbc/gel-foundations/spacings';
+import { RequestContext } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+import useToggle from '#hooks/useToggle';
+import socialEmbedBlockPropTypes from '#models/propTypes/socialEmbed';
+import createTranslations from './translations';
+
+const MAX_WIDTH = '31.25rem';
+
+const Wrapper = styled.div`
+  margin-right: auto;
+  margin-left: auto;
+  margin-bottom: ${GEL_SPACING_TRPL};
+  max-width: ${MAX_WIDTH};
+
+  .twitter-tweet,
+  .instagram-media {
+    margin-top: 0 !important;
+    margin-bottom: 0 !important;
+  }
+  .instagram-media {
+    min-width: auto !important;
+  }
+`;
+
+const SocialEmbedContainer = ({ blocks }) => {
+  const { isAmp } = useContext(RequestContext);
+  const { service, translations } = useContext(ServiceContext);
+  const { enabled } = useToggle('socialEmbed');
+
+  if (!blocks || !enabled) return null;
+
+  const { type: provider, model, id: uuid } = blocks[0];
+  const { id, href } = model;
+
+  const oEmbed = path(['embed', 'oembed'], model);
+
+  const {
+    fallback: fallbackTranslations,
+    skipLink: skipLinkTranslations,
+    caption: captionTranslations,
+  } = createTranslations(translations);
+
+  const fallback = {
+    ...fallbackTranslations,
+    linkHref: href,
+  };
+
+  const skipLink = {
+    ...skipLinkTranslations,
+    endTextId: `skip-%provider%-content-${uuid.split('-')[0]}`,
```

Comment:
Looks like this logic is specific to youtube embed ids e.g. https://github.com/bbc/simorgh/pull/5983/files#diff-ad71d8d6a3812135474f444b901b53c5R195 

Are you sure you won't encounter any collisions as you are spliting the unique id and only taking part of it? Maybe consder removing the hyphens instead?

Reply:
`uuid.split('-')[0]` splits the 128-bit unique block ID we get from Simorgh and takes the first 8 characters, which is a timestamp hash. I don't think a collision is possible in this context, however these ID's aren't 'friendly', so we're changing tack. These will become integers that identify embeds on the page in sequence – 1, 2, 3, etc.